pod/perlrun.pod

```diff
@@ -705,6 +705,12 @@ If the first character after the B<-M> or B<-m> is a dash (B<->)
 then the 'use' is replaced with 'no'.
 This makes no difference for B<-m>.
 
+Since Perl version 5.39.8, it is permitted to supply an empty C<-M> flag
+and the module name in the next positional argument instead.
```

Comment:
The wording sounds confusing to me, isn't the module name here being an argument to the `-M` option?

Here it sounds like `-M` is a boolean flag, and the module name is a positional argument to `perl` program proper, instead of the `-M` itself.

Reply:
Suggestion:

    Since Perl version 5.39.8, the B<-m>/B<-M> switch and the
    module name can be passed in separate command-line arguments:

    perl -M MODULE ...                # like perl -MMODULE ...
    perl -M MODULE=arg1,arg2,... ...  # like perl -MMODULE=arg1,arg2,... ...